sdk/ai/azure-ai-resources/dev_requirements.txt

```diff
@@ -0,0 +1,6 @@
+-e ../../../tools/azure-sdk-tools
+../../core/azure-core
+-e ../../../tools/azure-devtools
+../../identity/azure-identity
+pytest,
+pytest-xdist,
```

Comment:
I'm pretty certain that the pinned pytest in `test_tools.txt` will break this, but we'll see when the pipelines are actually added.

Reply:
Should I remove these then?

Reply:
FWIW, it's like this in `azure-ai-ml`'s `dev_requirements.txt`. Don't think it's causing any issues there